frontend/src/Components/RoleSelection.js

```diff
@@ -91,8 +92,8 @@ const RoleSelection = ({ question, matchedWith }) => {
       };
       const response = await axios.request(config);
       console.log(response.data);
-      //  if (role === "Mentor") {
-      //    navigate("/mentor-flow-1");
+      //  if (title === "Mentor") {
+      navigate("/mentor-flow-4");
```

Comment:
Not sure why we have this here in the get request

Reply:
this was just so we could attach the calendly page to the previous page i created so i could get access to the user's infomation, i was planning on changing this to navigate to the appropriate page once it is created since the industry page is still being worked on.